### ISSUE TITLE
The purpose of lb_ignore in LabelSmoothSoftmaxCE?

### ISSUE THREAD

**Issue by LightToYang** [CoinCheung/pytorch-loss#1]
It makes the loss ignored `idx=lb_ignore` in label ?

**Comment by CoinCheung** [comment]
Yes you are right, it should behave like pytorch F.nll_loss.

**Comment by CoinCheung** [comment]
Yes you are right, it should behave like pytorch F.nll_loss.
